.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: csharp
+solution: sdk-csharp.sln
+sudo: required
+dist: xenial
+mono: none
+dotnet: 2.2
+install:
+  - dotnet restore
+
+script:
+  - dotnet build Kuzzle/Kuzzle.csproj -c Release
+
+deploy:
+  skip_cleanup: true
+  provider: script
+  email: support@kuzzle.io
+  script:
+    - nuget pack Kuzzle/Kuzzle.nuspec -OutputFileNamesWithoutVersion
+    - nuget push kuzzlesdk.nupkg -ApiKey ${NUGET_API_KEY} -Source nuget.org
```

Comment:
Where is defined `NUGET_API_KEY` ?

Reply:
In travis' UI, as we do with other keys of that type (this way we don't have to generate and maintain an encrypted token in the travis file)

Reply:
(see https://travis-ci.org/kuzzleio/sdk-csharp/settings)